static/panes/compiler.js

```diff
@@ -925,6 +941,27 @@ Compiler.prototype.onIrViewClosed = function (id) {
     }
 };
 
+Compiler.prototype.onDeviceViewOpened = function (id) {
+    if (this.id === id) {
+        this.deviceButton.prop('disabled', true);
+        this.deviceViewOpen = true;
```

Comment:
If we're keeping this logic, can you initialise this to `false` in the constructor please?